pkg/ngrokapidriver/driver.go

```diff
@@ -22,25 +23,33 @@ type NgrokAPIDriver interface {
 	GetReservedDomains(ctx context.Context, edgeID string) ([]ngrok.ReservedDomain, error)
 }
 
+// NewNgrokAPIClient creates a new Driver setup with the passed in apiKey as well as default metadata
 type ngrokAPIDriver struct {
 	edges           *edge.Client
 	tgbs            *tgb.Client
 	routes          *edge_route.Client
 	reservedDomains *reserved_domains.Client
 	metadata        string
+	region          string
 }
 
-func NewNgrokApiClient(apiKey string) NgrokAPIDriver {
+func NewNgrokAPIClient(apiKey string, region string) NgrokAPIDriver {
```

Comment:
You can just do `apiKey, region string` for the function arguments.  That said, this is cool as is, I'm not sure if either is _more correct_ (but I am curious)